pkgs/tools/graphics/timg/default.nix

```diff
@@ -0,0 +1,52 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+, git
+, pkg-config
+, graphicsmagick
+, libjpeg
+, ffmpeg
+, zlib
+, libexif
+}:
+
+stdenv.mkDerivation rec {
+  pname = "timg";
+  version = "v1.4.0";
+
+  src = fetchFromGitHub{
+    owner = "hzeller";
+    repo = "timg";
+    rev = version;
+    sha256 = "10qhjfkbazncmj07y0a6cpmi7ki0l10qzpvi2zh8369yycqqxr8y";
+  };
+
+  buildInputs = [
+    graphicsmagick
+    ffmpeg
+    libexif
+    libjpeg
+    zlib
+  ];
+
+  nativeBuildInputs = [
+    cmake
+    git
```

Comment:
What is git used for at build-time?

Reply:
cmake rule attempts to extract version information if source is a git repository; it fails when it can't invoke git.

Added comment in default.nix for future reference.

Done.

Reply:
Fixed it in a better way now by telling `cmake` to not do that: ` -DTIMG_VERSION_FROM_GIT=Off`